src/arc.ts

```diff
@@ -134,12 +134,21 @@ export async function initializeArc(provider?: any): Promise<boolean> {
 
     // https://www.apollographql.com/docs/link/links/retry/
     const retryLink = new RetryLink({
```

Comment:
I think we can remove the retry link @roienatan

Reply:
Why?

Reply:
because the subscription already cover that in general.

Reply:
Note that `withSubscription` only handles Network and GraphQL errors (for now). If we're sure that those are the only possible errors so `retryLink` can be safely removed.

Reply:
other errors should be throw for now..so lets removed the retryLink.